src/Agent/NewRelic/Profiler/Common/Common.vcxproj.filters

```diff
@@ -6,5 +6,9 @@
     <ClInclude Include="Strings.h" />
     <ClInclude Include="OnDestruction.h" />
     <ClInclude Include="xplat.h" />
+    <ClInclude Include="AssemblyVersion.h" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="$(MSBuildThisFileDirectory)newrelic-icon.png" />
```

Comment:
hmm... do we want to include the icon?

Reply:
That's weird, I don't know how that got added.

Reply:
Removed